uploader.json

```diff
@@ -1932,6 +1932,13 @@
             "pattern": [],
             "symbolic_name": "zipl_conf"
         },
+        {
+            "file": "/etc/smart_proxy_dynflow_core/settings.yml",
+            "pattern": [
+		":database:"
+	    ],
+            "symbolic_name": "smartpdc_conf"
```

Comment:
@TripleJQK this looks good except that the symbolic_name does not match the name of the spec `smartpdc_settings`.  Please change the symbolic_name to match the name used in the spec.  Please do the same for `uploader.v2.json`.